templates/Notes.txt

```diff
@@ -1,6 +0,0 @@
-1. Remove glith_path from the session and instead call glitz_id in the url. 
-   This way we can use same page for multiple glitz id.
-   
-It's so frustrating to reboot the server after ever single change on template - anyone knows why? It wasn't the case earlier. 
```

Comment:
``app.debug = True`` should trigger this.  If you somehow turned this off, it will no longer reload.